ForceFeedback/ForceFeedback.Rules/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace ForceFeedback.Rules.Extensions
+{
+    public static class StringExtensions
```

Comment:
Code documentation is missing. Please describe the general intent of the class and methods.